js/ui/autocomplete.d.ts

```diff
@@ -75,13 +75,13 @@ export type OpenedEvent = EventInfo<dxAutocomplete>;
 export type OptionChangedEvent = EventInfo<dxAutocomplete> & ChangedOptionInfo;
 
 /** @public */
-export type PasteEvent = NativeEventInfo<dxAutocomplete>;
+export type PasteEvent = NativeEventInfo<dxAutocomplete, ClipboardEvent>;
 
 /** @public */
 export type SelectionChangedEvent = EventInfo<dxAutocomplete> & SelectionChangedInfo;
 
 /** @public */
-export type ValueChangedEvent = NativeEventInfo<dxAutocomplete> & ValueChangedInfo;
+export type ValueChangedEvent = NativeEventInfo<dxAutocomplete, KeyboardEvent | MouseEvent | PointerEvent | UIEvent> & ValueChangedInfo;
```

Comment:
```suggestion
export type ValueChangedEvent = NativeEventInfo<dxAutocomplete, KeyboardEvent | MouseEvent | PointerEvent | Event> & ValueChangedInfo;
```

For example, [change](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/change_event): `ChangeEvent: Event`

Depends on [valueChangeEvent ](https://js.devexpress.com/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/#valueChangeEvent)option